guides/common/modules/proc_configuring-the-project-client-in-keycloak.adoc

```diff
@@ -0,0 +1,23 @@
+[id="configuring-the-project-client-in-keycloak_{context}"]
+= Configuring the {Project} Client in {Keycloak}
+
+Use this procedure to configure the {Project} client in the {Keycloak} web UI and create group and audience mappers for the {Project} client.
+
+.Procedure
+
+. In the {Keycloak} web UI, navigate to *Clients* and click the {Project} client.
+. Ensure that the *Access Type* setting is set to *confidential*.
+. Locate the *Valid redirect URI* field that contains one redirect URI by default.
+Add a second *Valid redirect URI* in the following form: `https://{foreman-example-com}/users/extlogin`.
```

Comment:
This is a common mistake that I have seen with many community members, can we highlight this point in some way or we can mention that there must not be a slash in front of `https://{foreman-example-com}/users/extlogin`